users/test_routes.py

```diff
@@ -88,14 +97,131 @@ def test_missing_param(self):
+
+    def test_authorized_change_other_true_to_false(self):
+        """Authorized user can revoke authorization to other."""
+        target_user_id = AUTHORIZED_USER_ID
+        self.assertTrue(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+        self.client.post("/v1/authorization/update", data={
+            "target_user_id": target_user_id,
+            "is_organizer": False,
+            **DUMMY_GAUTH_REQUEST_DATA})
+        self.assertFalse(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+
+    def test_authorized_change_other_false_to_true(self):
+        """Authorized user can give authorization to other."""
+        target_user_id = NON_AUTHORIZED_USER_ID
+        self.assertFalse(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+        self.client.post("/v1/authorization/update", data={
+            "target_user_id": target_user_id,
+            "is_organizer": True,
+            **DUMMY_GAUTH_REQUEST_DATA})
+        self.assertTrue(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+
+    def test_authorized_change_other_true_to_true(self):
+        """Authorized user can trivially keep on authorization of other."""
+        target_user_id = AUTHORIZED_USER_ID
+        self.assertTrue(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+        self.client.post("/v1/authorization/update", data={
+            "target_user_id": target_user_id,
+            "is_organizer": True,
+            **DUMMY_GAUTH_REQUEST_DATA})
+        self.assertTrue(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+
+    def test_authorized_change_other_false_to_false(self):
+        """Authorized user can trivially keep off authorization of other."""
+        target_user_id = NON_AUTHORIZED_USER_ID
+        self.assertFalse(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+        self.client.post("/v1/authorization/update", data={
+            "target_user_id": target_user_id,
+            "is_organizer": False,
+            **DUMMY_GAUTH_REQUEST_DATA})
+        self.assertFalse(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+
+    def test_authorized_change_self_true_to_false(self):
+        """Authorized user can revoke authorization of self."""
+        target_user_id = UPDATER_ID
+        self.assertTrue(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+        self.client.post("/v1/authorization/update", data={
+            "target_user_id": target_user_id,
+            "is_organizer": False,
+            **DUMMY_GAUTH_REQUEST_DATA})
+        self.assertFalse(app.find_authorization_in_db(
+            target_user_id, app.app.config["COLLECTION"]))
+
+    def test_not_authorized(self):
```

Comment:
Maybe split this into separate tests? And create an unauthorized updater user so you don't have the change the updater's authorization to False.

Reply:
Done https://github.com/knative-portability/large-events/pull/41/commits/0e17603bf64465058b50e9767acb7f48bfff6235